README.md

```diff
@@ -0,0 +1,25 @@
+# Media service for MiSArch
+
+_Important: Set proper MinIO credentials before using the service._
```

Comment:
This might be a bit late, but as far as I remember, we decided against external dependencies. (This is why I didnt use stripe). 
Does the rest of the system work, if no credentials are set? (And there is just no possibility to upload images?)

Reply:
This is just a note that the user might want to change to username and password, as they are not secure for a real deployment. MinIO itself should not be an external dependency, as we are hosting our own instance using available source code: https://github.com/minio/minio, so its similar to Keycloak

Reply:
Ok then maybe make it a bit clearer, I E. "Consider setting secure credentials for production use"